README.md

```diff
@@ -257,3 +256,118 @@ We can now scale Kibana and Fluentd back up to their previous state.  Since Flue
+	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:165)
+	at com.floragunn.searchguard.filter.AbstractActionFilter.apply(AbstractActionFilter.java:105)
+	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:165)
+	at com.floragunn.searchguard.filter.AbstractActionFilter.apply(AbstractActionFilter.java:105)
+	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:165)
+	at org.elasticsearch.action.support.ActionFilter$Simple.apply(ActionFilter.java:64)
+	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:165)
+	at io.fabric8.elasticsearch.plugin.ActionForbiddenActionFilter.apply(ActionForbiddenActionFilter.java:48)
+	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:165)
+	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:82)
+	at org.elasticsearch.client.node.NodeClient.execute(NodeClient.java:98)
+	at org.elasticsearch.client.support.AbstractClient.get(AbstractClient.java:193)
+	at org.elasticsearch.action.get.GetRequestBuilder.doExecute(GetRequestBuilder.java:201)
+	at org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:91)
+	at com.floragunn.searchguard.service.SearchGuardConfigService.reloadConfig(SearchGuardConfigService.java:81)
+	at com.floragunn.searchguard.service.SearchGuardConfigService.access$500(SearchGuardConfigService.java:41)
+	at com.floragunn.searchguard.service.SearchGuardConfigService$Reload.run(SearchGuardConfigService.java:111)
+	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
+	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
+	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
+	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
+	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
+	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
+	at java.lang.Thread.run(Thread.java:745)
+```
+
+Since Fluentd and Kibana both talk to Elasticsearch, it is a good place to go for
+verifying that connections are active.
+
+You can see what indices have been created by Fluentd pushing logs to Elasticsearch:
+```
+[2016-02-19 21:01:53,867][INFO ][cluster.metadata         ] [M-Twins] [logging.2016.02.19] creating index, cause [auto(bulk api)], templates [], shards [5]/[1], mappings [fluentd]
+[2016-02-19 21:02:20,593][INFO ][cluster.metadata         ] [M-Twins] [.operations.2016.02.19] creating index, cause [auto(bulk api)], templates [], shards [5]/[1], mappings [fluentd]
+```
+
+And when a user logs in to Kibana, you will see their Kibana profile being created at the index `.kibana.{user-name sha hash}`.
```

Comment:
After a user signs into Kibana, their KIbana profile will have been created as the index ....